Shared/Extensions/StringExtensions.swift

```diff
@@ -109,4 +109,15 @@ public extension String {
         addWordsDescriptionBolded.setAttributes(attributes, range: nsRangeOfBoldedText)
         return addWordsDescriptionBolded
     }
+    
+    public func jsonObject() -> Any? {
+        do {
+            if let data = self.data(using: .utf8) {
+                return try JSONSerialization.jsonObject(with: data, options: .mutableContainers)
```

Comment:
I think you will be fine without the `.mutableContainers` since you are calling this function in swift. The containers are mutable by default.

Reply:
Updated thanks!